assets/dev/js/frontend/utils/swiper-bc.js

```diff
@@ -33,6 +33,9 @@ export default class SwiperBC {
 			legacySwiper();
 		}
 
+		// Fix: When adding margin to column of carousel widgets, it breaks the UI.
+		container.closest( '.elementor-widget-wrap' ).addClass( 'e-has-image-carousel-widget' );
```

Comment:
'e-has-image-carousel-widget' -> it's not just the image carousel, it can be for other widgets, I would call it something like: 'e-swiper-container' or at least something that is related to 'swiper'.